napari/_qt/_qapp_model/qactions/__init__.py

```diff
@@ -48,5 +49,5 @@ def _provide_qt_viewer() -> Optional[QtViewer]:
             return _qmainwin._qt_viewer
 
     # register actions
-    for action in chain(Q_VIEW_ACTIONS):
+    for action in chain(Q_VIEW_ACTIONS, Q_HELP_ACTIONS):
```

Comment:
if i understand this correctly, for each menu we’ve migrated to use app-model we’re gonna add to this chain?

Reply:
yes